File: module/visual-jdbc/src/main/java/org/visual/database/driver/Driver.java
/* (C)2024*/
package org.visual.database.driver;

import lombok.Getter;
import lombok.RequiredArgsConstructor;
import org.eclipse.aether.artifact.Artifact;
import org.eclipse.aether.artifact.DefaultArtifact;

@RequiredArgsConstructor
@Getter
public enum Driver {
    MYSQL(new DefaultArtifact("", "", "", "")),

    POSTGRESQL(new DefaultArtifact("", "", "", ""));

    private final Artifact artifact;
}

File: visual-model-database/src/main/kotlin/org/visual/model/sql/Driver.kt
package org.visual.model.sql

import org.eclipse.aether.artifact.Artifact
import org.eclipse.aether.artifact.DefaultArtifact

enum class Driver(val artifact: Artifact) {
    MYSQL(DefaultArtifact("", "", "", "")),

    POSTGRESQL(DefaultArtifact("", "", "", ""))
}